src/components/Button/Button.small.shame.scss

```diff
@@ -0,0 +1,40 @@
+// ************************************************************** //
```

Comment:
This whole commit removes the one-off scss in: `/src/components/UniversalNavbar/UniversalNavbar.scss` and I think the commit message says it all:

> Adds a one-off Button.Small only intended for use in the UniversalNavbar. I've made it as clear as possible that it's not to be used elsewhere. Removes UniversalNavbar.scss specific scss code and instead used the component directly--needed so we can use CSS Modules across the entire Button components

This was approved for time being by Design as I verified all this with Tyler. Someday we may want a `Button.Small`, but for now, this let's us proceed.